### ISSUE TITLE
Multiple header values

### ISSUE THREAD

**Issue by jdollenbacher** [MuleSoft-AI-Chain-Project/mulesoft-ai-chain-connector#32]
When working with "Tools use AI service" in the configuration JSON it is not possible to define multiple header values for the services used, e.g. "client_id" and "client_secret". This limits the usage as many APIs need multiple header values.

**Comment by amirkhan-ak-sf** [comment]
@jdollenbacher , yes thats a current limitation. I've added the support of multiple headers on our roadmap.
